### DISCUSSION TITLE
Love it!

### DISCUSSION THREAD

**Discussion by adamshand** [circled-me/server#10] [Feedback]
Hi,

Have just been playing with it for a couple of days and love the idea.  The UI is still a bit rough around the edges, but I can image as the project progresses that I could get my family to use it as a way to chat and share pictures instead of using Facebook.  Which would be amazing.

I don't have any experience with Go but am a reasonable JS/Svelte dev, if there's anything I can do to help I'd be interested.

Thanks!

**Comment by nikolaydimitrov** [comment]
Hi @adamshand 

Thanks for saying that. 
Actually that is what I’m using it for myself. Sharing photos with my family and friends. 
Also, almost finished the first version of the video/audio call functionality and already using it to talk to my family. It’s web-based so no app is even required to use it. 

Anyway, I would appreciate help on the design/UX/app/web sides though :)

Cheers

**Reply by adamshand** [reply]
Great, lemme play with it some more over the next weeks and see if I can get my wife using it to get some feedback from her as well.

**Reply by nikolaydimitrov** [reply]
Sounds good, let me know.

**Reply by adamshand** [reply]
Have just submitted a few minor bugs as I'm trying to get my wife setup.  If you'd rather them in a different format just let me know.

Thanks!
